src/ssh/setup.py

```diff
@@ -7,7 +7,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "1.0.0"
+VERSION = "1.0.1"
 
 CLASSIFIERS = [
     'Development Status :: 4 - Beta',
```

Comment:
We must remove beta here.

Reply:
@jiasli Now this extension is no longer in preview, what should the Development Status be set to? I looked at the Development Status of other extensions, and it looks like they are all set to "4 - Beta".

Reply:
I have no idea what it means. You may remove it.